stetho/src/main/java/com/facebook/stetho/inspector/database/DatabasePeerManager.java

```diff
@@ -107,17 +108,54 @@ private static String removeSuffix(String str, String[] suffixesToRemove) {
     Util.throwIfNull(handler);
     SQLiteDatabase database = openDatabase(databaseName);
     try {
-      Cursor cursor = database.rawQuery(query, null);
-      try {
-        return handler.handleResult(cursor);
-      } finally {
-        cursor.close();
+      String trimmedSql = query.trim();
+      String prefixSql = "";
+      if (trimmedSql.length() >= 3) {
+        prefixSql = trimmedSql.substring(0, 3).toUpperCase();
       }
+      if (prefixSql.equals("UPD") || prefixSql.equals("DEL")) {
```

Comment:
Also, curious why we don't spell out UPDATE, DELETE, etc?  Can you short-hand these in some way otherwise?